### PR TITLE
fix sphereLight normalization

### PR DESCRIPTION
**Pull request by elrond79**
were two issues:

1. it didn't take the radius into account
2. the formula for surface area was wrong the "3/" should have been inside
   the outermost pow, not outside

### CODE REVIEW

**Comment by bsavery** [comment]
I haven't run the math on this, but isn't there a 4p term mentioned in that link?  

`S   »   4p  [ ( a^pb^p + a^pc^p + b^pc^p ) / 3 ] ^ 1/p`

**Comment by elrond79** [comment]
There is, but we're just interested in the relative ratio:
surface_area(our_light) / surface_area(standard_light) - so any constant
terms will cancel out.  Or, put another way - I'm assuming we want to
standardize on a unit sphere, so any constant should be chosen such that a
sphere with radius = 1 will give a multiplicative factor of 1.

Though - come to think of it, I actually just assumed the "standard" was a
sphere with radius 1.  It might just as easily be a sphere with diameter
1... in which case we would need a corrective factor.  Is there a standard
consensus on this in the rendering community at large?

Interesting note: on the page I linked,
http://www.numericana.com/answer/ellipsoid.htm, it uses 4*p, whereas on
wikipedia (https://en.wikipedia.org/wiki/Ellipsoid#Approximate_formula), it
uses 4*pi... Not sure which one is correct, but again, for our use case, it
doesn't matter...

**Comment by elrond79** [comment]
Just for completeness - cross referencing with the surface area of a sphere
- 4*pi*r^2 - the correct factor is 4*pi, not 4*p.  Wikipedia wins!

**Comment by bsavery** [comment]
Yeah I was wondering about the pi factor as well.  Seemed strange there wasn't one in there ;) . 

Unless I'm misreading USDLux the default radius is 1 btw .

File: pxr/imaging/plugin/hdRpr/sphereLight.cpp [hunk 1]
```diff
@@ -53,14 +53,17 @@ RprApiObject HdRprSphereLight::CreateLightMesh(std::map<TfToken, float> & params
...
 	constexpr const double p = 1.6075;
 	constexpr const double pinv = 1. / 1.6075;
+	const float radius = params[HdRprSphereLightTokens->radius];
 
-	const double sx = GfVec3d(transform[0][0], transform[1][0], transform[2][0]).GetLength();
-	const double sy = GfVec3d(transform[0][1], transform[1][1], transform[2][1]).GetLength();
-	const double sz = GfVec3d(transform[0][2], transform[1][2], transform[2][2]).GetLength();
+	const double sx = GfVec3d(transform[0][0], transform[1][0], transform[2][0]).GetLength() * radius;
+	const double sy = GfVec3d(transform[0][1], transform[1][1], transform[2][1]).GetLength() * radius;
+	const double sz = GfVec3d(transform[0][2], transform[1][2], transform[2][2]).GetLength() * radius;
```
Comment (bsavery) [inline pxr/imaging/plugin/hdRpr/sphereLight.cpp]: BTW we could also add an optimization here of course if SX == SY == SZ not a huge deal though

Comment (elrond79) [reply pxr/imaging/plugin/hdRpr/sphereLight.cpp]: Hmm - yeah, the approximation is exact when SX == SY == SZ, so we could switch over to the optimization without worrying about any "jump"...

**Pull request merged**